types/tokenizr/index.d.ts

```diff
@@ -0,0 +1,264 @@
+// Type definitions for tokenizr 1.5
+// Project: https://github.com/rse/tokenizr
+// Definitions by: Nicholas Sorokin <https://github.com/aNickzz>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export {};
+
+export class Tokenizr {
```

Comment:
It should be like this:

```ts
class Tokenizr { ... }

export = Tokenizr
```

```ts
import Tokenizr = require('tokenizr');

const lexer = new Tokenizr();
```

https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default